### ISSUE TITLE
Is there anybody out there?

### ISSUE THREAD

**Issue by tomasaschan** [neoeinstein/TickSpec#2]
Yo! Not sure if there is any activity here at the moment, but I'm cross-posting from [codeplex](https://tickspec.codeplex.com/discussions/662907) just to maximize my hopes of somebody reading it.

Not much has happened with the source code for this library in a long time, it seems, but I'm interested in porting it - or building something that works very similarly to it - to .NET Core.

I tried just naively copying the source files into a new .NET Core solution and fixing all the compiler errors. Most of it actually worked quite fine, but two pieces of code in particular made a working port more difficult to create than I had hoped. Both are related to this issue: https://github.com/dotnet/coreclr/issues/10199

The TickSpec code in question:

* In `FeatureGen.fs`, `ModuleBuilder.DefineDocument` is missing: https://github.com/neoeinstein/TickSpec/blob/89cc7963bf1c4d9df33dfb4dab706292af4a5e61/TickSpec/FeatureGen.fs#L34
* In `ScenarioGen.fs`, `ILGenerator.MarkSequencePoint` is missing: https://github.com/neoeinstein/TickSpec/blob/89cc7963bf1c4d9df33dfb4dab706292af4a5e61/TickSpec/ScenarioGen.fs#L335

I don't know enough about the inner workings to know how to work around this; perhaps it's possible to utilize the F# compiler services for generating the code, and avoid having to emit IL altogether?

It would be great to come in contact with someone who has some deeper knowledge about how this package works, to help make it survive into the next generation of the .NET and F# ecosystem.

**Comment by tomasaschan** [comment]
Pinging @ptrelford as I don't know if you follow this repo, but think you'd be able to help me out here if you like. After all, it seems like you wrote the thing :)

**Comment by bartelink** [comment]
This repo looks like a clone of mine at https://github.com/bartelink/tickspec (which is good as it fixes history relative to the hg version on codeplex). I'd like to see my xunit v2 updates (and gitignore etc) be propagated to wherever this goes if at all possible.

**Comment by tomasaschan** [comment]
@bartelink Definitely. It wouldn't surprise me if there are more clones, too, but if I do start some serious work on this I definitely want to support xUnit 2, and not incorporating your fixes would just mean double work :)
